Vault-Deployment/values/promtail.yaml

```diff
@@ -0,0 +1,23 @@
+config:
+  clients:
+    - url: http://loki.loki.svc.cluster.local:3100/loki/api/v1/push
+      # Send everything to the kubernetes tenant_id by default
+      # https://grafana.com/docs/loki/latest/operations/multi-tenancy
+      tenant_id: promtail
+  snippets:
+    pipelineStages:
+      # https://grafana.com/docs/loki/latest/send-data/promtail/stages/cri
+      # https://github.com/grafana/helm-charts/blob/main/charts/promtail/values.yaml#L427
+    - cri: {}
+    - match:
+        # https://support.hashicorp.com/hc/en-us/articles/360000995548-Audit-and-Operational-Log-Details
+        # https://grafana.com/docs/loki/latest/send-data/promtail/stages/match
+        # Drop anything that does not start with a timestamp
+        #selector: '{app="vault" |~ ̣"^{.*"}'
+        selector: '{app="vault"}'
+        stages:
+        - drop:
+            expression: '{\\"time.*'
+      # Remove all vault server and audit log lines
+      #action: drop
+      #drop_counter_reason: vault_audit_match
```

Comment:
if you enable these lines it would remove all Vault logs (server and audit logs)